### PR TITLE
Issue 85 increase sidecar memory

### LINKED ISSUE #85: Increase memory allocation for consul sidecar proxy
## Feature description
Increase memory allocation for minio sidecar proxy
## Why is it needed?
Increase memory allocation
## Definition of done
Side car proxy services are running with more memory
## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by karimacherradi**
## Closes/fixes/resolves issue(s)?
- closes #85 
## Checklist (after created PR)
- [ ] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

**Comment by zhenik** [comment]
Could you update: 
- README.md, input section, and example section
- in `example/minio_standalone/main.tf` setup `proxy_resource` variables explicit

File: example/minio_standalone/main.tf [hunk 1]
```diff
@@ -22,7 +22,8 @@ module "minio" {
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
   use_canary                      = true
-
+  cpu_proxy                       = 200
+  memory_proxy                    = 128
```
Comment (zhenik) [inline example/minio_standalone/main.tf]: I think it should be
```suggestion
resource_proxy = {
  cpu = 200
  memory = 128
}
```

File: README.md [hunk 2]
```diff
@@ -106,6 +106,8 @@ module "minio" {
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
   use_canary                      = true
+  cpu_proxy                       = 200
+  memory_proxy                    = 128
```
Comment (zhenik) [inline README.md]:
```suggestion
resource_proxy = {
  cpu = 200
  memory = 128
}
```

File: README.md [hunk 3]
```diff
@@ -142,6 +144,7 @@ module "minio" {
 | vault_secret.vault_kv_access_key | Secret key name in Vault kv path | string | "access_key" |
 | vault_secret.vault_kv_secret_key | Secret key name in Vault kv path | string | "secret_key" |
 | minio\_upstreams | List up connect upstreams | list(object) | [] | no |
+| resource_proxy | Resource allocations for proxy | obj(number,number) | {cpu = 200,memory = 128} | no |
```
Comment (zhenik) [inline README.md]: Linter :) 
```suggestion
| resource_proxy | Resource allocations for proxy | obj(number,number) | {cpu = 200,memory = 128} | no |
```

**Review by zhenik — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,9 @@
 
 ### Added
 - Added skip-duplicate-runs #78
+- Added sidecar_task #85
+- Modified expose for healthchecks #85
+
```
Comment (claesgill) [inline CHANGELOG.md:9]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (claesgill) [inline README.md:147]:
```suggestion
| resource_proxy | Resource allocations for proxy | obj(number,number) | { <br> cpu = 200, <br> memory = 128 <br> } | no |
```
`<br>` makes it more readable.

Comment (claesgill) [reply README.md]:
```suggestion
| resource_proxy | Resource allocations for proxy | obj(number,number) | { <br> cpu = 200, <br> memory = 128 <br> } | no |
```
`<br>` makes it more readable.

File: main.tf [hunk 5]
```diff
@@ -49,7 +49,8 @@ data "template_file" "nomad_job_minio" {
     use_host_volume       = var.use_host_volume
     use_canary            = var.use_canary
     upstreams             = jsonencode(var.minio_upstreams)
-
+    cpu_proxy                 = var.resource_proxy.cpu
+    memory_proxy              = var.resource_proxy.memory
```
Comment (claesgill) [inline main.tf]: Equal spacing, please 😄

Comment (karimacherradi) [reply main.tf]: Okey :P

File: example/minio_standalone/main.tf [hunk 6]
```diff
@@ -22,7 +22,10 @@ module "minio" {
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
   use_canary                      = true
-
+resource_proxy = {
+  cpu = 200
+  memory = 128
+}
```
Comment (pdmthorsrud) [inline example/minio_standalone/main.tf]: Should be indented. Should probably check that I put the right amount of spaces if you commit this suggestion!
```suggestion
  resource_proxy = {
    cpu = 200
    memory = 128
  }
```

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]
LGMT

**Review by claesgill — APPROVED** [review]
# 🚀

**Review by dangernil — APPROVED** [review]

**Pull request merged**